ThermofluidStream/Undirected/Topology/ConnectForeFore.mo

```diff
@@ -1,7 +1,7 @@
 within ThermofluidStream.Undirected.Topology;
 model ConnectForeFore "Connects two fore ports"
 
-  extends ThermofluidStream.Utilities.DropOfCommonsPlus;
+  extends ThermofluidStream.Utilities.DropOfCommonsPlus(displayInstanceName=false);
```

Comment:
Why this change?

Reply:
I suggested to change the default value to false, because i had the feeling that `ConnectForeFore` is one of the last models where user would want to display the instance name.
Actually currently displayInstanceName=false for **every** example in the TFS. We might first consider setting `displayInstanceName=true` for at least a few important components in the TFS examples (like source/sink, heat exchangers, compressors).

Reply:
> I suggested to change the default value to false, because i had the feeling that ConnectForeFore is ...

As is for `ConnectForeOutlet` & Co. as well.

> We might first consider setting displayInstanceName=true for at least a few important components ...

This is in fact a subjective choice. Hope this will be helpful to users.